components/calendar/src/types.rs

```diff
@@ -137,6 +137,20 @@ impl MonthCode {
         }
         None
     }
+
+    /// Construct a "normal" month code given a number ("Mxx").
+    ///
+    /// Returns an error for months greater than 99
+    pub fn new_normal(number: u8) -> Option<Self> {
```

Comment:
Thought: This is new public API. We should probably bikeshed. But it seems okay.

Reply:
yeah I wasn't sure about it either but I've reached for this kind of thing before.

I made it `cfg(test)` for now. I didn't feel strongly about it. We can discuss separately.